hivemind_bus_client/util.py

```
@@ -221,6 +128,62 @@ def bytes2str(payload: bytes, compressed=False) -> str:
+        "decrypt_from_json is deprecated and will be removed in future versions. "
+        "Use 'from hivemind_bus_client.encryption import decrypt_from_json' instead",
+        DeprecationWarning,
+        stacklevel=2
+    )
+    from hivemind_bus_client.encryption import decrypt_from_json as _dj, JsonCiphers
+    try:
+        return _dj(key, data, JsonCiphers.JSON_HEX_AES_GCM_128)
+    except Exception as e:
+        try:
+            return _dj(key, data, JsonCiphers.JSON_B64_AES_GCM_128)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)